region/norfair/west/Hi Jump Energy Tank Room.json

```diff
@@ -240,10 +338,69 @@
         {"or": [
           "canWalljump",
           "h_canFly",
-          "canUseFrozenEnemies"
+          "canUseFrozenEnemies",
+          {"and": [
+            "HiJump",
+            "canSpringBallJumpMidAir"
+          ]}
         ]}
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode through Bomb Blocks",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphMovement",
+        "canOffScreenMovement"
+      ],
+      "note": "Overload PLMs using scroll blocks in the morph tunnel. The camera will not scroll to the left side of the room."
+    },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Overload PLMs with the Left Item",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": true
+        }
+      },
+      "requires": [
+        {"itemNotCollectedAtNode": 4},
+        "canRiskPermanentLossOfAccess",
+        "h_canArtificialMorphPowerBomb",
+        "canOffScreenMovement"
+      ],
+      "note": [
+        "Placing a Power Bomb a 0.5 to 1 tile away from the item plinth to bounce Samus onto it while also overloading PLMs.",
```

Comment:
I don't understand.  Are you landing on the Etank?  Does this require the Etank to be there to use 1 Power bomb?